### DISCUSSION TITLE
https://maxkim-j.github.io/posts/js-pure-function

### DISCUSSION THREAD

**Discussion by MaxKim-J** [MaxKim-J/maxkim-j.github.io#91] [Announcements]
#          📥 부수효과 없는 "순수함수" 맛보기      

https://maxkim-j.github.io/posts/js-pure-function

**Comment by MaxKim-J** [comment]
<p>낼름낼름 지식 냠냠</p>

>_This comment was made by Disqus. 2021-04-28 07:43:05 라이언(disqus_Bam85l2bkM)_

**Comment by lygggg** [comment]
재미있게 읽었습니다! (꾸벅)
